.gitignore

```diff
@@ -8,6 +8,7 @@
 
 # Secrets
 .env
```

Comment:
One thing of note, doesn't have to be fixed here but might should be a new issue - we use a `.env` file in the `fxa-payments-server` server as well as the `fxa-admin-panel` package. It's for local dev but we might not want this at the root-level, I had to force add it when making `-panel`. This is how we're proxying an express server to the webpack dev server.

Reply:
Just want to make sure I understand - your suggestion is to not use a single .env at the root level, and instead use multiple at the package level to avoid issues with server config?

Reply:
I agree that, at least for now, the `.env` files should be per service since it matches how we're configuring them now. In the future I'd like to consider using a single config for all services.

Reply:
Makes sense. Reminder that we're not tied to .env - this could be a JSON/JS file if we prefer. I believe .env assumes you could pass variables in when starting the server, which is not the case here, it's simply just parsing the text content of the env file.